lib/racer-client.coffee

```diff
@@ -17,10 +17,14 @@ class RacerClient
       cb null
       return
 
+    editor_file_path = editor.getPath().substring(0, )
+    editor_folder_path = editor_file_path.substring(0, editor_file_path.lastIndexOf("/"))
```

Comment:
Hum, I'm not very sure about using substrings and index of a "/" to get a base path.
Maybe it would be better to use something from the `path` module like `path.basename()` ?